docs/ref/models/expressions.txt

```diff
@@ -450,6 +450,88 @@ Conditional expressions allow you to use :keyword:`if` ... :keyword:`elif` ...
 :keyword:`else` logic in queries. Django natively supports SQL ``CASE``
 expressions. For more details see :doc:`conditional-expressions`.
 
+``SubQuery`` expressions
+------------------------
+
+.. class:: SubQuery(queryset, output_field=None)
+
+.. versionadded:: 1.11
+
+When a value is required that is dependent upon values from within a related
+object, it's possible to use an explicit ``SubQuery``. For example::
+
+    >>> largest = Size.objects.filter(relation=OuterRef('pk')).order_by('-size')
+    >>> queryset.annotate(largest_size=SubQuery(largest.values('size')[:1]))
+
+.. note::
```

Comment:
`.. note::` usage seems excessive -- maybe section headings are needed or at least (`.. admonition:: descriptive title`)?  If you keep any notes, please use 4 space indent.

Please wrap documentation (and comments/docstrings in code) at 79 chars.